src/CMakeLists.txt

```diff
@@ -115,9 +115,17 @@ set (RUNTIME_BC
   compute_35
 )
 
+# Replace escaped space with a simple space ('\ ' -> ' ')
```

Comment:
How does an escaped space get into this var? What happens if we don't unescape it?